server/service/src/standard_reports.rs

```diff
@@ -0,0 +1,88 @@
+use repository::{
+    ContextType, FormSchemaJson, FormSchemaRowRepository, ReportRow, ReportRowRepository,
+    StorageConnection, StorageConnectionManager,
+};
+use rust_embed::RustEmbed;
+use thiserror::Error;
+
+use crate::report::definition::ReportDefinition;
+use log::info;
+use serde::{Deserialize, Serialize};
+
+#[derive(RustEmbed)]
+// Relative to server/Cargo.toml
+#[folder = "../reports/generated"]
+
+pub struct EmbeddedStandardReports;
+
+#[derive(Debug, Error)]
+#[error("No standard reports found")]
+pub struct StandardReportsError;
+
+#[derive(Clone)]
+pub struct StandardReports;
+
+impl StandardReports {
+    // Load embedded reports
+    pub fn load_reports(connection_manager: StorageConnectionManager) -> Result<(), anyhow::Error> {
+        let con = connection_manager.connection()?;
+
+        for file in EmbeddedStandardReports::iter() {
+            if let Some(content) = EmbeddedStandardReports::get(&file) {
+                let json_data = content.data;
+                let reports_data: ReportsData = serde_json::from_slice(&json_data)?;
+                let _ = StandardReports::upsert_reports(reports_data, &con)?;
+            }
+        }
```

Comment:
Would be a nice to have, but would like a log message saying how many reports have been inserted...